src/index.common.js

```diff
@@ -9,9 +9,23 @@ const ngModuleName = 'formly';
 const angular = require('./angular-fix');
 const ngModule = angular.module(ngModuleName, []);
 
-require('./providers')(ngModule);
-require('./services')(ngModule);
-require('./directives')(ngModule);
-require('./run')(ngModule);
+ngModule.constant('formlyApiCheck', require('./providers/formlyApiCheck'));
```

Comment:
Is there a reason we can't use `import` here? I realize that you can't do an inline `import` statement like you can with `require` so we'd have to put all of these imports at the top, which is annoying, but I think it'd be nice to be consistent.